### ISSUE TITLE
update browser versions

### ISSUE THREAD

**Issue by smanders** [smanders/buildpro#45]
firefox and chrome browsers were added to centos7-run https://github.com/smanders/buildpro/issues/21 and chrome browser added to centos7-bld https://github.com/smanders/buildpro/issues/30 in previous commits/issues

we have permission to update our minimum browser compatibility
firefox: 102.6.0
chrome: 108.0.5359.99
https://msbtrac.sdl.usu.edu/ticket/16699

**Comment by smanders** [comment]
CommonLibraries/Angular/.browserslistrc https://isrhub.usurf.usu.edu/Vantage/CommonLibraries/blob/7131ae4c7d5be080f9794ee4296afe92d4a01e14/Angular/.browserslistrc is a good place to verify the minimum browser version, which currently is
```
Firefox >= 102
Chrome >= 108
```

**Comment by smanders** [comment]
current buildpro chrome version: https://dl.google.com/linux/chrome/rpm/stable/x86_64/google-chrome-stable-94.0.4606.81-1.x86_64.rpm (fails to download)
searching UbuntuUpdates for the version 108 https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable?id=202706&page=4
latest chrome 108 version `108.0.5359.124`
https://dl.google.com/linux/chrome/rpm/stable/x86_64/google-chrome-stable-108.0.5359.124-1.x86_64.rpm

**Comment by smanders** [comment]
current firefox version: `78.15.0esr`
available versions https://ftp.mozilla.org/pub/firefox/releases/
latest 102 version availabe: `102.13.0esr`

**Comment by smanders** [comment]
so automated tests in (unit tests, etc) that are run in a bld container can also run in firefox, we also want to add firefox to the bld image

**Comment by smanders** [comment]
completed with commits referenced above

verified updates in bld container
```
smanders@bpbldlatest:/bpvol/buildpro (dev)
$ google-chrome --version
Google Chrome 108.0.5359.124 

smanders@bpbldlatest:/bpvol/buildpro (dev)
$ firefox --version
Mozilla Firefox 102.13.0esr

```

**Comment by smanders** [comment]
haha, I didn't even read my own notes!
versions should be
firefox: 102.6.0
chrome: 108.0.5359.99

**Comment by smanders** [comment]
chrome 108.0.5359.99 isn't available but 108.0.5359.98 is
https://dl.google.com/linux/chrome/rpm/stable/x86_64/google-chrome-stable-108.0.5359.98-1.x86_64.rpm

**Comment by smanders** [comment]
completed with commits referenced above

verified updates in bld container

```
smanders@bpbldlatest:/bpvol/buildpro (dev)
$ google-chrome --version 
Google Chrome 108.0.5359.98 

smanders@bpbldlatest:/bpvol/buildpro (dev)
$ firefox --version
Mozilla Firefox 102.6.0esr

```
